@AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageDTUDraw.sqf

```diff
@@ -4,4 +4,11 @@ params ["_heli", "_mpdIndex", "_state"];
 #include "\fza_ah64_dms\headers\constants.h"
 
 _heli setUserMfdValue [MFD_INDEX_OFFSET(MFD_IND_DTU_SUBPAGE), _state get "subPage"];
-_heli setUserMfdValue [MFD_INDEX_OFFSET(MFD_IND_DTU_LOAD_STATUS), _state get "loadState"]
\ No newline at end of file
+_heli setUserMfdValue [MFD_INDEX_OFFSET(MFD_IND_DTU_LOAD_STATUS), _state get "loadState"];
+
+date params ["_year", "_month", "_day", "_hours", "_minutes"];
+private _dtuDate = str _month + "/" + str _day + "/" + ((str _year) select [2,2]);
```

Comment:
While I don't see any direct examples for dates, other uses of times show that padding `0`s.

Could you use `fza_fnc_padString` for this?